libp2p/builders.nim

```diff
@@ -64,7 +64,8 @@ proc new*(T: type[SwitchBuilder]): T =
     maxOut: -1,
     maxConnsPerPeer: MaxConnectionsPerPeer,
     protoVersion: ProtoVersion,
-    agentVersion: AgentVersion)
+    agentVersion: AgentVersion,
+  )
```

Comment:
Yaiks! can we not do the new line in this kinda calls and istead use the old style:

```nim
agentVersion: AgentVersion)
```

This triggers my OCD every time :) 

(unless this is a new addition to the style guide...)

Reply:
It's just badly-reverted changes from a50bc81, mb